api/v1beta1/flowcollector_types.go

```diff
@@ -89,24 +89,25 @@ type FlowCollectorSpec struct {
 // allow defining both fields.
 // +union
 type FlowCollectorAgent struct {
-	// type selects the flows tracing agent. Possible values are "EBPF" (default) to use NetObserv eBPF agent,
-	// "IPFIX" - <i>deprecated (*)</i> - to use the legacy IPFIX collector. "EBPF" is recommended in most cases as it offers better
-	// performances and should work regardless of the CNI installed on the cluster.
-	// "IPFIX" works with OVN-Kubernetes CNI (other CNIs could work if they support exporting IPFIX,
+	// type selects the flows tracing agent. Possible values are:<br>
+	// - `EBPF` (default) to use NetObserv eBPF agent.<br>
+	// - `IPFIX` - <i>deprecated (*)</i> - to use the legacy IPFIX collector.<br>
```

Comment:
this one was ambiguous as it was deprecated before even our 1st GA ... but considering the history of the project upstream I would say it's deprecated, since it was once the only available option